providers/aws/resources/aws_dynamodb.go

```diff
@@ -102,14 +102,17 @@ func initDynamoDbTable(runtime *plugin.Runtime, args map[string]*llx.RawData) (m
 	}
 	dynamodb := obj.(*mqlAwsDynamodb)
 
-	rawResources, err := dynamodb.tables()
+	rawResources := dynamodb.GetTables()
 	if err != nil {
 		return nil, nil, err
 	}
```

Comment:
This is now checking an error that has been checked before and can be removed I think